test/UniversalRewardsDistributorTest.sol

```diff
@@ -334,15 +413,6 @@ contract UniversalRewardsDistributorTest is Test {
         distributionWithTimeLock.acceptRoot();
     }
 
-    function testSetTimelockShouldRevertIfNotOwner(uint256 newTimelock, address randomCaller) public {
-        vm.assume(randomCaller != owner);
-        newTimelock = bound(newTimelock, 0, type(uint256).max);
-
-        vm.prank(randomCaller);
-        vm.expectRevert(bytes(ErrorsLib.NOT_OWNER));
-        distributionWithoutTimeLock.setTimelock(newTimelock);
-    }
```

Comment:
Why is this test removed? It is still part of the specs, it's just missing `vm.assume(newTimelock != TIMELOCK)`